README.md

```diff
@@ -1,9 +1,14 @@
 # realtime_r0_brazil
 
-## Métodos para estimar o número de reprodução eficaz no Brasil
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
 
-Depois de conter a primeira onda de COVID-19, fazer projeções vai ficar cada vez mais difícil, já que teremos mudanças de regime frequentes (vamos entrar na fase da "dança" do the [The Hammer and the Dance](https://medium.com/@tomaspueyo/coronavirus-the-hammer-and-the-dance-be9337092b56)). Dessa forma, devemos passar para uma estratégia de acompanhamento e controle, e para isso, precisamos de uma boa métrica.
+## Methods to estimate the effective breeding number in Brazil
 
-Acreditamos que o número de reprodução eficaz Rt é a resposta. Inspirados pelo artigo do [Kevin Systrom](http://systrom.com/blog/the-metric-we-need-to-manage-covid-19/), reproduzimos o modelo do artigo [neste notebook](https://github.com/gdmarmerola/realtime_r0_brazil/blob/master/realtime_r0_brazil.ipynb) e devemos propor outros no futuro.
+After containing the first wave of COVID-19, making projections will become increasingly difficult, as we will have frequent regime changes (we will enter the "dance" phase of the [The Hammer and the Dance](https://medium.com/@tomaspueyo/coronavirus-the-hammer-and-the-dance-be9337092b56)). Thus, we must move on to a monitoring and control strategy, and for that, we need a good metric.
 
-Sugestões são bem-vindas!!!!
+We believe that the effective breeding number Rt is the answer. Inspired by the [Kevin Systrom](http://systrom.com/blog/the-metric-we-need-to-manage-covid-19/) article, we reproduce the model of the article on the notebook present on this repository and we must propose others in the future.
```

Comment:
I don't know what is the right notebook, but I think it will be cool do put the link here.

Reply:
yeah, the link was there before, but it was wrong. The original repo contained a file named `realtime_r0_brazil.ipynb`, but the current one is named `realtime_r0_bettencourt_ribeiro.ipynb`, so I assumed that it may change again in the future.
One alternative is to rename it back to `realtime_r0_brazil.ipynb`, wdyt?

Reply:
I did the alternative, and renamed the notebook (then I could reference it again here in the readme), wdyt?